tests/e2e.test.js

```diff
@@ -95,25 +95,22 @@ describe('External links', () => {
     await page.goto(url.href);
   });
 
-  it('is possible to click the link to GitHub', async () => {
-    await expect(page).toClick('a', { text: 'GitHub' });
-  });
-
-  it('is possible to click the link to NPM', async () => {
-    await expect(page).toClick('a', { text: 'NPM' });
-  });
-
-  it('is possible to click the link to Packagist', async () => {
-    await expect(page).toClick('a', { text: 'Packagist' });
-  });
-
-  it('is possible to click the JSDelivr link', async () => {
-    await expect(page).toClick('a', { text: 'JSDelivr' });
-  });
-
-  it('is possible to click the Unpkg link', async () => {
-    await expect(page).toClick('a', { text: 'Unpkg' });
-  });
+  const menuLinks = [
+    'repository',
+    'npm',
+    'packagist',
+    'jsDelivr (CDN)',
+    'Unpkg (CDN)',
+    'Open Collective',
+    'Legal disclaimer',
+    'About Simple Icons',
+  ];
+
+  menuLinks.forEach((title) =>
```

Comment:
Love the improvement 👍 Just one suggestion for clarity:

```suggestion
  const menuLinksTitles = [
    'repository',
    'npm',
    'packagist',
    'jsDelivr (CDN)',
    'Unpkg (CDN)',
    'Open Collective',
    'Legal disclaimer',
    'About Simple Icons',
  ];

  menuLinksTitles.forEach((title) =>
```